server/api/router.go

```diff
@@ -220,6 +220,10 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) *mux.R
 	apiRouter.Handle("/metric/query", newQueryMetric(svr)).Methods("GET", "POST")
 	apiRouter.Handle("/metric/query_range", newQueryMetric(svr)).Methods("GET", "POST")
 
+	// tso API
+	tsoHandler := newTSOHandler(svr, rd)
+	apiRouter.HandleFunc("/tso/allocator/transfer/{name}", tsoHandler.TransferLocalTSOAllocator).Methods("POST")
```

Comment:
local_allocator?

Reply:
I think point out the `local_allcaotor` here is not necessary.